src/utils/storage.ts

```diff
@@ -1,54 +1,40 @@
-import { OpenWeatherTempScale } from './api'
+import uniqid from "uniqid";
+import { generateRandomColor } from "./colors";
+import { NoteType } from "../types/noteType";
 
-export interface LocalStorage {
-  cities?: string[]
-  options?: LocalStorageOptions
-}
+export const clearStorage = () => {};
 
-export interface LocalStorageOptions {
-  hasAutoOverlay: boolean
-  homeCity: string
-  tempScale: OpenWeatherTempScale
-}
+export const fetchNotes = async (url: string): Promise<any[]> => {
+  return new Promise((resolve) => {
+    chrome.storage.sync.get([url], (notes) => {
+      resolve(notes[url] ? JSON.parse(notes[url]) : []);
+    });
+  });
```

Comment:
why do we need to use Promise here?

Reply:
Copy from tutorial, maybe we don't need it here, will check

Reply:
UPD: Promise helps to return data from a callback